source/content.css

```diff
@@ -851,3 +851,27 @@ body > .footer li a {
 .markdown-body details:not([open]) { /* Summary-less details have unselectable clickable elements */
 	cursor: pointer;
 }
+
+/* For 'add-toggle-files-button' feature */
+.js-toggle-files {
+	margin-right: -6px;
+}
+.commit-tease .octicon-chevron-up {
+	display: none;
+}
+.files-hidden .commit-tease {
+	border-bottom-left-radius: 3px;
+	border-bottom-right-radius: 3px;
```

Comment:
This can probably just be a single `border-radius: 3px;`

Reply:
You're right.